node_agent.spec

```diff
@@ -0,0 +1,67 @@
+%define pkg_name tendrl-node-agent
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Common Module for All Bridges
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: LGPL2.1
```

Comment:
This `License:` field should be one of the values from https://fedoraproject.org/wiki/Licensing:Main

For example, "License: LGPLv2"